src/ts/component/block/text.tsx

```diff
@@ -643,7 +643,7 @@ const BlockText = observer(class BlockText extends React.Component<Props> {
 		e.persist();
 
 		const { onKeyDown, rootId, block, isInsideTable } = this.props;
-		const { id } = block;
+		const { id,content:{text} } = block;
```

Comment:
I do not see where it's used and we do not write this way, `const { text } = block.content;` for example.

Reply:
Sorry, I forgot to delete some test codes. The original intention was to try automatic indentation.